syft/frameworks/torch/hook/hook.py

```diff
@@ -27,6 +27,7 @@
 
 from syft.exceptions import route_method_exception
 from syft.exceptions import TensorsNotCollocatedException
+from torch._six import inf
```

Comment:
Is there a better place to import inf from? six is mostly geared for 2:3 compatability

Reply:
I replaced it with 'from math import inf'. I suppose that should work too?